internal/config/validate.go

```diff
@@ -327,9 +327,15 @@ func (c *Pipeline) Validate() error {
 	var result error
 
 	for _, stepRaw := range c.StepRaw {
-		if stepRaw == nil || stepRaw.Use == nil || stepRaw.Use.Type == "" {
+		if stepRaw == nil && stepRaw.PipelineRaw == nil {
```

Comment:
The second part of this conditional would never get hit, since if `stepRaw == nil` then it would never evaluate `stepRaw.PipelineRaw`.
```suggestion
		if stepRaw == nil || stepRaw.PipelineRaw == nil {
```

Reply:
I think I'll want to split up the if-statement then because if `stepRaw` is nil, `stepRaw.PipelineRaw` will panic. But makes sense!